graspy/utils/utils.py

```diff
@@ -656,3 +656,59 @@ def cartprod(*arrays):
     return np.transpose(
         np.meshgrid(*arrays, indexing="ij"), np.roll(np.arange(N + 1), -1)
     ).reshape(-1, N)
+
+
+def fit_plug_in_variance_estimator(X):
```

Comment:
@bdpedigo reminder to think about whether this should be a function or a class and what to call it

Reply:
and how should it work. returning a function is flexible, but might not be intuitive to many people.

Reply:
closing here; see #378.